lumibot/backtesting/backtesting_broker.py

```diff
@@ -484,21 +448,25 @@ def process_pending_orders(self, strategy):
                 # This is a hack to get around the fact that we need to get the previous day's data to prevent lookahead bias.
                 ohlc = strategy.get_historical_prices(
                     asset,
-                    1,
+                    2,
                     quote=order.quote,
-                    timeshift=-1,
+                    timeshift=-2,
```

Comment:
I'm confused about the timeshift update.  The use case for this is that you need more than one candle right because a asset is thinly traded, right?  In that case, you don't need to do any timeshift, instead you just need more (potential) candles.  Am I missing something?

Reply:
Resolved in live chat